src/pages/basic_data/BasicDataPage.tsx

```diff
@@ -60,6 +60,7 @@ const BasicDataPage = () => {
         minimizedMenu={
           <Link
             to={{
+              // TODO: Ensure this points to something sensible when on import pages
```

Comment:
Mulig vi må sette `state: { previousPath }` rundt omkring, og bruke den her med feks `pathname: location.state.previousPath ?? 'central-import`.
Men da må vi nok også passe på at `previousSearch` tas vare på alle plasser det trengs eller kan trengs i fremtiden 🤔 

Mulig det blir nor sånt flere plasser: 
```
to={{
  pathname: "xxx",
  state: { ...currentState, previousPath: "yyy" } // ...currentState for å ta vare på søket
}}
```
Aner meg at dette ikke blir en helt åpenbar flyt, og knot med debugging om vi går for det... :/ 
Må først avklare hvor man skal sendes i alle situasjoner her.